ts/src/base/Exchange.ts

```diff
@@ -2737,10 +2737,10 @@ export default class Exchange {
 
     afterConstruct () {
         this.createNetworksByIdObject ();
-        this.featuresGenerator ();
+        this.generateFeatures ();
     }
 
-    featuresGenerator () {
+    generateFeatures () {
         //
```

Comment:
only cosmetical rename, to help users IDEs to avoid suggesting these methods that start with `feature...`. so i've renamed them to somewhat uninterested names.